src/test/groovy/com/jvm_bloggers/core/data_fetching/blogs/json_data/JsonBlogsDataFileValidationSpec.groovy

```diff
@@ -73,5 +65,37 @@ class JsonBlogsDataFileValidationSpec extends Specification {
         assert duplicateIds.isEmpty() : "Duplicate bookmarkable IDs found in json files: $duplicateIds"
     }
 
-}
+    @Unroll
```

Comment:
If you use `@Unroll`, you should also use `#parameterName` in test method name so we will have more granular names there.

Reply:
Done.